src/templates/pycontw-2020/contents/en/registration/financial-aid.html

```diff
@@ -0,0 +1,45 @@
+{% extends 'contents/_base.html' %}
+
+{% block title %}Financial Aid{% endblock title %}
+
+{% block meta_description %}Overview on how to submit a financial aid application to PyCon Taiwan, and allow us to help you attend the conference.{% endblock meta_description %}
+
+
+{% block content %}
+
+<h1>Financial Aid</h1>
+
+<h2>The Program</h2>
+
+<p><a href="http://jessenoller.com/blog/2011/05/25/pycon-everybody-pays" target="_blank" rel="noopener">Everybody Pays</a> is one of PyCon’s core principles. All attendees including speakers and volunteers have to pay for registering. While following this convention, PyCon Taiwan also has a financial aid program to help friends in the community, financially or otherwise, so more people are able join us in the big event.</p>
```

Comment:
We no longer use "Everybody pays". Instead, we use "Everybody contributes". See PyCon TW 2020 CFP.